Tests/xSharePoint/xSharePoint.xSPWebApplication.Throttling.Tests.ps1

```diff
@@ -0,0 +1,244 @@
+[CmdletBinding()]
+param(
+    [string] $SharePointCmdletModule = (Join-Path "C:\users\brian\Documents\GitHubVisualStudio\xSharePoint\Tests\xSharePoint" "..\Stubs\SharePoint\15.0.4693.1000\Microsoft.SharePoint.PowerShell.psm1" -Resolve)
```

Comment:
Hello. I also got that in my last commit. Perhaps you should sync before pushing ?



> On 1 Dec 2015, at 8:40 AM, Brian Farnhill <notifications@github.com> wrote:
> 
> In Tests/xSharePoint/xSharePoint.xSPWebApplication.Throttling.Tests.ps1:
> 
> > @@ -0,0 +1,244 @@
> > +[CmdletBinding()]
> > +param(
> > +    [string] $SharePointCmdletModule = (Join-Path "C:\users\brian\Documents\GitHubVisualStudio\xSharePoint\Tests\xSharePoint" "..\Stubs\SharePoint\15.0.4693.1000\Microsoft.SharePoint.PowerShell.psm1" -Resolve)
> Thats my bad - have fixed it in the next PR :P
> 
> —
> Reply to this email directly or view it on GitHub.
>